index-proteomics.md

```diff
@@ -3,19 +3,84 @@ layout: subsite-galaxy
+
+Server | Tutorial(s) | Video
+--- | --- | ---
+[Proteogenomics Gateway](http://z.umn.edu/proteogenomicsgateway){:target="_blank"} | [Proteogenomics 1: Database creation](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html){:target="_blank"}<br/>[Proteogenomics 2: Database search](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html){:target="_blank"}<br/>[Proteogenomics 3: Novel peptide analysis](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html){:target="_blank"} | [![proteogenomics thumbnail](https://img.youtube.com/vi/Xwks5qKApj8/0.jpg)](https://www.youtube.com/watch?v=Xwks5qKApj8&t=4s){:target="_blank"}
+[Metaproteomics Gateway](http://z.umn.edu/metaproteomicsgateway){:target="_blank"} | [Metaproteomics tutorial](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/metaproteomics/tutorial.html){:target="_blank"} | [![metaproteomics thumbnail](https://img.youtube.com/vi/Vywt_-D5Kek/0.jpg)](https://www.youtube.com/watch?v=Vywt_-D5Kek&t=1s){:target="_blank"}
+{: .table.table-striped .tooltable}
+
+<br/>
+
+# Projects
+
+Featured Galaxy-P tools supported with manuscripts, visualization plugins, docker containers, and hands-on tutorials.
+
+Tool | Description | Paper | Tutorial | Docker
+--- | --- | --- | --- | ---
+CRAVAT-P | Stands for Cancer Related Analysis of Variants Toolkit for Proteogenomics. Submits, checks for, and retrieves data for cancer annotation | [Sajulga et. al 2018](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00404){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.8b00404/suppl_file/pr8b00404_si_001.pdf){:target="_blank"} | [![docker-icon](https://cdn3.iconfinder.com/data/icons/social-media-2169/24/social_media_social_media_logo_docker-128.png)](http://z.umn.edu/gpcravatdocker){:target="_blank"}
+QuanTP | Correlation between protein and transcript | [Kumar et. al 2019](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00727){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.8b00404/suppl_file/pr8b00404_si_001.pdf){:target="_blank"} | [![docker-icon](https://cdn3.iconfinder.com/data/icons/social-media-2169/24/social_media_social_media_logo_docker-128.png)](http://z.umn.edu/quantpdocker){:target="_blank"}
+MetaQuantome | Quantitative analysis of the function and taxonomy of microbiomes and their interaction |  | [<i class="fa fa-magic" aria-hidden="true"></i>](https://galaxyproteomics.github.io/metaquantome_mcp_analysis/galaxy_tutorial/tutorial.html){:target="_blank"} |
+{: .table.table-striped .tooltable}
+
+<br/>
+
+# News and Events
```

Comment:
If you think the events and the carousel is too much, please feel free to remove them.

Reply:
Okay. I will ask the other Galaxy-P members about this issue.

Reply:
It's not an issue I just wanted to say that if we feel like this is too much content please remove it. I just added it to have some initial content.

Reply:
Okay. I like having it on the page, so you're right: definitely not an issue.